src/tsickle.ts

```diff
@@ -931,7 +931,7 @@ class Annotator extends ClosureRewriter {
 
     let tags = this.getJSDoc(p) || [];
     tags.push({tagName: 'type', type: this.typeToClosure(p)});
-    this.emit(jsdoc.toString(tags));
+    this.emit(jsdoc.toString(tags, ['param']));
```

Comment:
Can you add a comment explaining why we need to filter `@param` here? That'll make it easier to add more when it comes up.